e2e_test/source/basic/inlcude_key_as.slt

```diff
@@ -59,5 +75,25 @@ select count(rw_key) from upsert_students_default_key
 ----
 15
 
+query I
+SELECT count(*)
+FROM   additional_columns
+WHERE  key_col IS NOT NULL
+       AND partition_col IS NOT NULL
+       AND offset_col IS NOT NULL
+       AND timestamp_col IS NOT NULL
+       AND header_col IS NOT NULL
+----
+101
+
+query TT
+SELECT (header_col[1]).key AS key, (header_col[1]).value::text AS value
+FROM   additional_columns limit 1;
+----
+header1 \x7631
```

Comment:
Can we leave a docstring here to explain this header value?

Reply:
No, Kafka header is a list of kvs, with schema (varchar, bytes). The list can be empty, ie. no header and multiple kvs,

here is an example

https://github.com/risingwavelabs/risingwave/blob/600300a31babec3f4956f088b4f8788cdd104714/scripts/source/prepare_ci_kafka.sh#L79

kcat will generate 101 messages containing header with two kvs (header1, v1) and (header2, v2).

Reply:
added